worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/regions/plotsquared/PlotSquaredFeature.java

```diff
@@ -60,7 +60,7 @@ public PlotSquaredFeature() {
     }
 
     public static String getName(UUID uuid) {
-        return UUIDHandler.getName(uuid);
+        return PlayerManager.getName(uuid);
     }
```

Comment:
Unneeded to keep, it's unused and a blocking operation.

Reply:
Can you suggest a replacement for this code? PlotSquared has changed significantly since I last worked on it.

Reply:
Like I said, it's not used anywhere, hence unneeded.

Reply:
Misunderstood. Gotcha.